src/index.css

```diff
@@ -72,25 +72,46 @@ svg {
   margin-right: 0.25rem;
 }
 
+table {
+  border-collapse: collapse;
+  width: 100%;
+}
+th,
+td {
+  padding: 8px 16px;
+}
+
 .tableFixHead {
   overflow-y: auto;
   max-height: 18em;
 }
+
+.tableFixHead:first-of-type {
+  margin-bottom: 0.67em;
+}
+
 #waypoints thead th {
   position: sticky;
   top: 0;
 }
 
-table {
-  border-collapse: collapse;
-  width: 100%;
+#waypoints-head th,
```

Comment:
Related to "Refactored table rows coloring to have dark odd and light even rows"